src/perl/lib/wtsi_clarity/epp/generic/stamper.pm

```diff
@@ -31,6 +32,52 @@ has 'step_url' => (
   required   => 1,
 );
 
+override 'run' => sub {
```

Comment:
Please, do not rearrange the file content, because it is really hard to find what is/are the real change(s).